pkg/plugins/builtin/http/http.go

```diff
@@ -29,7 +29,7 @@ type HTTPConfig struct {
 	Body           string      `json:"body,omitempty"`
 	Headers        []Header    `json:"headers,omitempty"`
 	TimeoutSeconds string      `json:"timeout_seconds,omitempty"`
-	HTTPBasicAuth  BasicAuth   `json:"basic_auth,omitempty"`
+	Auth           Auth        `json:"auth,omitempty"` // FIXME how to manage deprecation of the basic auth attribute ?
```

Comment:
I'm not _too_ afraid or slightly breaking changes right now because the installed user base is small -- that being said, I think long term the best way to deal with such breaking changes is to document them really well in the changelog.
Since the version upgrade is an involved action from the utask end user (updating his Dockerfile FROM, or even re-running the new install script, or make update, or something), displaying this clearly should help fix the affected templates.

Reply:
fine, documentation has been updated